package/yast2.spec

```diff
@@ -52,6 +52,7 @@ BuildRequires:  yast2-core >= 2.18.12
 BuildRequires:  yast2-devtools >= 3.1.10
 # Pkg.Resolvables()
 BuildRequires:  yast2-pkg-bindings >= 4.2.0
+BuildRequires:  yast2-packager
```

Comment:
Warning: circular dependency: https://github.com/yast/yast-packager/blob/master/package/yast2-packager.spec#L39. I think you should omit it or move the needed classes (from `Y2Packager`) to this repository.